js/client.js

```diff
@@ -1722,18 +1723,25 @@ function toId() {
 			if (room.requestLeave && !room.requestLeave(e)) return false;
 			return this.removeRoom(id);
 		},
-		renameRoom: function (id, newid) {
+		renameRoom: function (id, newtitle, newid) {
+			var newid = newid || toRoomid(newtitle);
```

Comment:
And `|| toRoomid(newtitle)` should be dropped; it should instead be `newtitle = newtitle || newid`.